_posts/2016-05-29-new-web-site.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post-no-cover
+title: Prepare a new website of the CESE method
+date: 2016-05-29 07:20:00 +0800
+categories: posts
+show: true
```

Comment:
As default.html fixes the side bar order as your above change, the "show" tag is not used anymore.
Please feel free to remove this customized tag.